website/docs/cli-reference.md

```diff
@@ -22,6 +22,20 @@ Disable validation of hints when building the contracts.
 Migrate project sources to Cairo 0.10.
 #### `targets STRING[]=['.']`
 Targets to migrate (a target can be a file or directory)
+### `calculate-account-address`
+In order to create an account, you need to prefund the account. To prefund the account you need to know its address. This command calculates the account address.
+#### `--account-address-salt INT`
```

Comment:
Consistency with the `deploy_account` command.

Reply:
Why to include them in `deploy_account` in the first place? it is a lot to write :p
No need to change though I agree it is better to leave them for the sake of consistency

Reply:
hmm indeed, what are they used for in `deploy_account`? this command shouldn't only get precomputed account address?

Reply:
Deploy account sends a “deploy account” transaction, and these arguments must be included in the transaction.

Reply:
@kasperski95 yess, I meant prefixes, not arguments (sorry I indeed formulated my sentence in a weird way).

Reply:
@MaksymilianDemitraszek I know you referred to the prefixes. That was more an answer to the @mkaput's question.

To calculate the address, you may also need the deployer address. I used account prefixes to distinguish account from deployer. I didn't expose deployer address argument because I couldn't find an information why and when is needed.

Reply:
Merging. If those prefixes are an issue, we need to deprecate them first in a different PR.